File: springboot/spring-boot/src/main/java/com/example/demo/entity/Post.java
package com.example.demo.entity;

import lombok.*;

import javax.persistence.*;
import java.util.Date;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Post {

    @GeneratedValue(strategy=GenerationType.IDENTITY)
    @Id
    @Column(name = "postId")
    private int postId;
    private int typeId;
    private int userId;
    private String userName;
    private String content;
    private int imageNum;
    private int commentCount;
    private int likeCount;
    private int dislikeCount;
    private int time;

}

File: app/src/main/java/com/example/anonymous_community_app/data/Post.kt
package com.example.anonymous_community_app.data

data class Post(
    val postId: Int = 0,
    val typeId: Int = 0,
    val userId: Int = 0,
    val userName: String = "",
    val content: String = "",
    val imageNum: Int = 0,
    val commentCount: Int = 0,
    val likeCount: Int = 0,
    val dislikeCount: Int = 0,
    val time: Int = 0
)
